src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -425,6 +445,18 @@ export default {
 				&& !this.isModerator
 				&& !this.isGroup
 		},
+		canBeAllowedPublishing() {
+			return this.participant.publishingAllowed !== PARTICIPANT.PUBLISHING_ALLOWED.ALWAYS
+				&& this.selfIsFullModerator
+		},
+		canBeDisallowedPublishing() {
+			return this.participant.publishingAllowed !== PARTICIPANT.PUBLISHING_ALLOWED.NEVER
+				&& this.selfIsFullModerator
```

Comment:
This would be a good usecase for guest moderators from my POV. Any reason you wouldn't allow it?

Reply:
In the original hack there was an issue with guest moderators (it was not possible to differentiate between guests and guests moderators, I do not remember why), so only moderators but not guest moderators could publish. I think that I subconsciously dragged that here. I will enable it for guest moderators too.

Reply:
Then the `@PublicPage` change above is not needed but you need to adjust the second annotation on that endpoint :)